scipy/stats/tests/test_distributions.py

```diff
@@ -3496,7 +3496,7 @@ def test_gh_pr_10949_argcheck(self):
 
 class TestRayleigh(object):
     def setup_method(self):
-        np.random.seed(1234)
+        np.random.seed(987654321)
```

Comment:
Just curious why?

Reply:
It was that seed and the values `[0.74, 0.01]` in the test `test_fit_comparison_super_method` that reliably triggered the test failure of the old code.